src/queueloader.cpp

```diff
@@ -214,6 +215,20 @@ void QueueLoader::reload(std::vector<Download>& downloads, bool remove_unplayed)
 		f.close();
 	}
 
+	for (const auto& dl : deletion_list) {
+		std::string filename = dl.filename();
+		if (access(filename.c_str(), F_OK) == 0) {
+			LOG(Level::INFO,
+				"Deleting file %s",
+				filename);
+			if (std::remove(filename.c_str()) != 0) {
+				LOG(Level::ERROR,
+					"Failed to delete file %s",
+					filename);
```

Comment:
It'd be useful to log `errno` here, ideally both as an integer and as text.

Reply:
Good idea, done